### PR TITLE
add invisible Watermark

### PR DESCRIPTION
**Pull request by Songyuanwei**

### CODE REVIEW

File: examples/stable_diffusion_v2/tools/watermark/invisible_watermark/dwtDctSvd.py
Comment (SamitHuang) [inline examples/stable_diffusion_v2/tools/watermark/invisible_watermark/dwtDctSvd.py:1]: keep file naming in under_score_case

Comment (SamitHuang) [reply examples/stable_diffusion_v2/tools/watermark/invisible_watermark/dwtDctSvd.py:1]: same for the other two py files

**Comment by SamitHuang** [comment]
Please supplement the README document

File: examples/stable_diffusion_v2/tools/watermark/invisible_watermark/dwtDctSvd.py
Comment (Songyuanwei) [reply examples/stable_diffusion_v2/tools/watermark/invisible_watermark/dwtDctSvd.py:1]: done

**Review by zhtmike — APPROVED** [review]

File: examples/stable_diffusion_v2/tools/watermark/maxDct.py
Comment (SamitHuang) [inline examples/stable_diffusion_v2/tools/watermark/maxDct.py:1]: `maxDct.py` -> `max_dct.py`

File: examples/stable_diffusion_v2/tools/watermark/dwtDctSvd.py
Comment (SamitHuang) [inline examples/stable_diffusion_v2/tools/watermark/dwtDctSvd.py:1]: dwtDctSvd.py -> dwt_dct_svd.py

File: examples/stable_diffusion_v2/tools/watermark/maxDct.py
Comment (Songyuanwei) [reply examples/stable_diffusion_v2/tools/watermark/maxDct.py:1]: fixed

File: examples/stable_diffusion_v2/tools/watermark/dwtDctSvd.py
Comment (Songyuanwei) [reply examples/stable_diffusion_v2/tools/watermark/dwtDctSvd.py:1]: fixed

**Review by SamitHuang — APPROVED** [review]

**Pull request merged**